docs/02_installation/README.md

```diff
@@ -88,16 +88,16 @@ Nevertheless, you don't need to be a rocket scientist to install EasyBuild (and
 that's OK too), so don't worry.
 
 You can install EasyBuild just like you can install any other Python software that is released
-via the standard *Python Package Index* (PyPI), through one of standard Python installation tools
-(did you catch the irony in that last bit?). And since EasyBuild is a software installation tool
+via the standard *Python Package Index* (PyPI), through one of standard Python installation tools.
+There may be some irony involved for this specific solution, and since EasyBuild is a software installation tool
```

Comment:
What solution are you referring to here? Seems confusing to me now...

Reply:
That was a poor attempt at removing parenthesis. :-) Redone in the coming push.